code/modules/admin/holder2.dm

```diff
@@ -5,6 +5,7 @@ GLOBAL_PROTECT(href_token)
 
 /datum/admins
 	var/rank			= "Temporary Admin"
+	var/list/sranks		= null
```

Comment:
Would be great to have a name somewhat more descriptive, all of that stuff is already cryptic as hell